pkg/apis/storage/types.go

```diff
@@ -280,8 +280,12 @@ type CSIDriverSpec struct {
 	// Defines if the underlying volume supports changing ownership and
 	// permission of the volume before being mounted.
 	// Refer to the specific FSGroupPolicy values for additional details.
-	// This field is alpha-level, and is only honored by servers
+	// This field is beta, and is only honored by servers
 	// that enable the CSIVolumeFSGroupPolicy feature gate.
+	// Defaults to ReadWriteOnceWithFSType, which will examine each volume
+	// to determine if Kubernetes should modify ownership and permissions of the volume.
+	// If a fstype is defined and the volume's access mode
+	// contains ReadWriteOnce, then the defined fsGroup will be applied.
```

Comment:
I initially interpreted this to mean that fsgrouppolicy only applies if it's rwo. Can we make it more clear that this criteria only applies to the RWOWithFstype value?

Reply:
This is currently in the description of the `ReadWriteOnceWithFSTypeFSGroupPolicy`. I've reworded it here to make it more explicit that this only applies to the default policy.